planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/scene.cpp

```diff
@@ -1631,41 +1665,6 @@ bool NormalLaneChange::getLaneChangePaths(
           continue;
         }
 
-        if (
-          lane_change_parameters_->regulate_on_crosswalk &&
-          !hasEnoughLengthToCrosswalk(*candidate_path, current_lanes)) {
-          if (getStopTime() < lane_change_parameters_->stop_time_threshold) {
-            debug_print_lat("Reject: including crosswalk!!");
-            continue;
-          }
-          RCLCPP_INFO_THROTTLE(
-            logger_, clock_, 1000, "Stop time is over threshold. Allow lane change in crosswalk.");
-        }
-
-        if (
-          lane_change_parameters_->regulate_on_intersection &&
-          !hasEnoughLengthToIntersection(*candidate_path, current_lanes)) {
-          if (getStopTime() < lane_change_parameters_->stop_time_threshold) {
-            debug_print_lat("Reject: including intersection!!");
-            continue;
-          }
-          RCLCPP_WARN_STREAM(
-            logger_, "Stop time is over threshold. Allow lane change in intersection.");
-        }
-
-        if (
-          lane_change_parameters_->regulate_on_traffic_light &&
-          !hasEnoughLengthToTrafficLight(*candidate_path, current_lanes)) {
-          debug_print_lat("Reject: regulate on traffic light!!");
-          continue;
-        }
-
-        if (utils::traffic_light::isStoppedAtRedTrafficLightWithinDistance(
-              get_current_lanes(), candidate_path.value().path, planner_data_,
-              lane_change_info.length.sum())) {
-          debug_print_lat("Ego is stopping near traffic light. Do not allow lane change");
-          continue;
-        }
```

Comment:
@zulfaqar-azmi-t4 can you confirm that its okay to remove these checks, knowing that I modified the activation condition [here](https://github.com/autowarefoundation/autoware.universe/pull/8648/files#diff-59cbf60b192ecddbbd9709016c0b4bd8846eda2ddbc91dc28937762391e13ff0R198) to use the maximum possible lane change length.